source/tutorial/backend.txt

```diff
@@ -53,22 +53,28 @@ You'll need to have the following before you can begin this tutorial:
   for MongoDB Atlas and create one for free
   <https://account.mongodb.com/account/register?tck=docs_realm>`__.
 
+Also note that this tutorial doesn't use the Deployment Draft workflow. Unless you have disabled 
+Deployment Drafts, you may need to save your changes to a draft and then deploy them manually.
+
 Create a New {+app+}
 ----------------
 
-First, you'll need to create an application in {+service-short+}. To create the app:
+First, you'll need to create an application in {+service-short+}. If you see a prompt about 
+Aplication Guides, select :guilabel:`Close Guides`.
```

Comment:
i think the sentence `If you see a prompt about Application Guides, select Close Guides`.

Reply:
Thanks for questioning this. As I thought about it, I agree that it's unnecessary. I've removed it.